bang/forms.py

```diff
@@ -123,14 +123,15 @@ class Meta(AutoForm.Meta):
         save_owner_on_creation = True
 
 class MemberFilterForm(MagiFiltersForm):
-    search_fields = ['name', 'japanese_name', 'school', 'CV', 'romaji_CV', 'food_like', 'food_dislike', 'instrument', 'hobbies', 'description']
+    search_fields = ['name', 'japanese_name', 'school', 'CV', 'romaji_CV', 'classroom', 'height', 'food_like', 'food_dislike', 'instrument', 'hobbies', 'description']
```

Comment:
I don't think anyone's gonna try to enter 160 hoping to find all characters with 160cm height?